synapse/storage/databases/main/event_federation.py

```diff
@@ -73,13 +73,20 @@
 
 logger = logging.getLogger(__name__)
 
-BACKFILL_EVENT_BACKOFF_UPPER_BOUND_SECONDS: int = int(
-    datetime.timedelta(days=7).total_seconds()
-)
+BACKFILL_EVENT_EXPONENTIAL_BACKOFF_MAXIMUM_DOUBLING_STEPS = 8
 BACKFILL_EVENT_EXPONENTIAL_BACKOFF_STEP_SECONDS: int = int(
     datetime.timedelta(hours=1).total_seconds()
 )
```

Comment:
Isn't this just 3600? Honestly I'd find it clearer if it was just:

```python
# One hour.
BACKFILL_EVENT_EXPONENTIAL_BACKOFF_STEP_SECONDS = 3600
```

Reply:
No strong opinions. FWIW I didn't want to overrule the choices made in #13635.

Reply:
The plain English 1 hour to to seconds makes more sense to me

We could use `BACKFILL_EVENT_EXPONENTIAL_BACKOFF_STEP_MS = 1000 * ...` if constant milliseconds is preferred (mentioned in https://github.com/matrix-org/synapse/pull/13936#discussion_r982673530)